File: ModularitatClasses/graella.py
class Coordenada:
    def __init__(self):
        # Coordenada inicial
        self.x: int = 0
        self.y: int = 0

    def moure_dreta(self):
        self.x += 1

    def moure_esquerra(self):
        self.x -= 1

    def moure_amunt(self):
        self.y += 1

    def moure_avall(self):
        self.y -= 1

class Main:
    # Programa principal
    def __init__(self):
        # Executar moviments
        coordenada: Coordenada = Coordenada()
        coordenada.moure_dreta()
        print(f"Nova coordenada després de moure a la dreta: ({coordenada.x}, {coordenada.y})")

        coordenada.moure_amunt()
        print(f"Nova coordenada després de moure amunt: ({coordenada.x}, {coordenada.y})")

Main()

File: UF2-Programacio_Modular/ModularitatClassesKotlin/graella.kt
class Coordenada
{
		var x: Int
		var y: Int

		constructor()
		{
			// Coordenada inicial
			x = 0
			y = 0
		}

		fun moure_dreta()
		{
			x += 1
		}

		fun moure_esquerra()
		{
			x -= 1
		}

		fun moure_amunt()
		{
			y += 1
		}

		fun moure_avall()
		{
			y -= 1
		}
	}

	class Main
	{
		// Programa principal
		constructor()
		{
			// Executar moviments
			var coordenada: Coordenada = Coordenada()
			coordenada.moure_dreta()
			println("Nova coordenada després de moure a la dreta: (${coordenada.x}, ${coordenada.y})")

			coordenada.moure_amunt()
			println("Nova coordenada després de moure amunt: (${coordenada.x}, ${coordenada.y})")
		}
	}

	fun main()
	{
		Main()
	}
